DOCS-1947: English/grammar check on Lightspeed app page (#1011)
MultiSafepay/docs: content/integrations/hosted/lightspeed_app/faq/maximal-order-amount.md

meta_title: "Lightspeed plugin - Can I set a maximun order value for a payment method to show up? - MultiSafepay Docs"
meta_description: "The MultiSafepay Documentation Center presents all relevant information about our Plugins and API. You can also find support pages for payment methods, tools and general questions as well as the contact details of our Support and Integration Teams."
---

> Yes, you can change the maximum order amount for a payment method to be shown in the checkout. To do so, you have to open the payment method in the _Payment methods ordering_ list by clicking on the "+" sign. You will then be required to enter an amount in EUR cents in the maximum field. If you do not want a maximum amount enter “-1”.

Reviewer: ```suggestion
Yes, you can change the maximum order amount for a payment method to be shown in the checkout. To do so, you have to open the payment method in the _Payment methods ordering_ list by clicking on the "+" sign. Next, enter an amount in EUR cents in the maximum field. If you do not want a maximum amount, enter “-1”.
```
